src/files.rs

```diff
@@ -62,8 +63,7 @@ mod tests {
 
     #[test]
     fn file_only_works_on_files() {
```

Comment:
Would `cant_use_include_file_on_directories` be a better name?

Reply:
I think this discussion is discussion is valid, but not an essential part of this PR.

Naming is part of the higher arts of testing. To tell the truth I never read the test name. I was on this file because the other test broke. Removing the content parts left me in the mood to remove more.

I would suggest to avoid negations in names and aim for the shortest name. `reject_directory` or `err_on_directory`. This also makes it obvious that a test for non existing files is missing. On the other hand this code now is so trivial, that I would argue that you have enough tests as is.

The more interesting discussion is if this file is still needed. Basically File is a PathBuf. PathBuf allows to check the file existence in one line. So we have a file with 65 lines of code that gain us basically nothing for the rest of this crate.

Reply:
You raise a good point. I'll create a separate issue regarding the test though seeing as it isn't really relevant to the PR.